ImperatorToCK3/Source/CK3/CK3World.cpp

```diff
@@ -25,7 +25,10 @@ CK3::World::World(const Imperator::World& impWorld, const Configuration& theConf
 	// Loading vanilla CK3 landed titles
 	landedTitles.loadTitles(theConfiguration.getCK3Path() + "/game/common/landed_titles/00_landed_titles.txt");
 	// Loading regions
-	regionMapper = std::make_unique<mappers::CK3RegionMapper>(theConfiguration.getCK3Path(), landedTitles);
+	ck3RegionMapper = std::make_shared<mappers::CK3RegionMapper>(theConfiguration.getCK3Path(), landedTitles);
+	imperatorRegionMapper = mappers::ImperatorRegionMapper(theConfiguration.getImperatorPath());
+	// Use the region mappers in other mappers
+	religionMapper.loadRegionMappers(std::make_shared<mappers::ImperatorRegionMapper>(imperatorRegionMapper), ck3RegionMapper);
```

Comment:
just pass the pointer and target function will copy it. you can std::move incoming copy over there. That's what shared_ptr is for.

Reply:
To make sure, you're suggesting this, right? https://clang.llvm.org/extra/clang-tidy/checks/modernize-pass-by-value.html

Reply:
yah.
imperatorRegionMapper = std::make_shared<mappers::ImperatorRegionMapper>(theConfiguration.getImperatorPath());
religionMapper.loadRegionMappers(imperatorMapper);
Then when you pass it, it'll copy the pointer.

Reply:
github ate my braces.